dynawo/sources/Models/CPP/ModelNetwork/DYNModelGenerator.h

```diff
@@ -382,8 +382,17 @@ class ModelGenerator : public NetworkComponent {
   std::weak_ptr<GeneratorInterface> generator_;  ///< reference to the generator interface object
   double Pc_;  ///< active power target in MW
   double Qc_;  ///< reactive power target in Mvar
+  double P0_;  ///< initial active power
+  double Q0_;  ///< initial reactive power
+  double u0_;  ///< initial voltage
+  double U0Pu_square_;
   double ir0_;  ///< initial current real part
   double ii0_;  ///< initial current imaginary part
+  double alpha_;  ///< active power exponential sensitivity to voltage
+  double beta_;  ///< reactive power exponential sensitivity to voltage
+  double halfAlpha_;
+  double halfBeta_;
+  bool isVoltageDependant_;  ///< whether the produced energy remains constant
```

Comment:
```suggestion
  bool isVoltageDependant_;  ///< false if the produced energy remains constant
```
Also add descriptions for other parameters?